ARS_simulation.py

```diff
@@ -236,9 +237,9 @@ def simulate_workload_random(function: str):
         return True
 
 
-pr_lock = threading.Lock()
-number_of_parallel_requests_pending = 0
-startedCommands = {}
+pr_lock = multiprocessing.Lock()
+number_of_parallel_requests_pending = multiprocessing.Value('i', 0)
+startedCommands = multiprocessing.Manager().dict({})
```

Comment:
dieser Aufruf sieht seltsam aus. Wenn ich es richtig verstanden habe, muss der Manager explizit in einer Variablen gespeichert werden, sonst wird der Manager vom GC aufgeräumt. Weiterhin wird so ein Manager-Objekt und ein Dictionary-Objekt pro Prozess erzeugt. Siehe https://stackoverflow.com/a/9536888

Oder ist das hier eine Besonderheit von guvicorn in Verbindung mit dem --preload Flag?